experiments/compositions/composition/release/kind/operator.yaml

```diff
@@ -1125,10 +1125,10 @@ spec:
         - --health-probe-bind-address=:8081
         - --metrics-bind-address=127.0.0.1:8080
         - --leader-elect
-        - --image-registry=gcr.io/krmapihosting-release
```

Comment:
Do we actually want to merge this files changes?

Reply:
reverted.

Reply:
Still showing up in the PR.

Reply:
Once https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1776/ is merged. I will rebase. This file will be removed.